Passwordless.sln

```diff
@@ -29,6 +29,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "examples", "examples", "{6E
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Passwordless.Example", "examples\Passwordless.Example\Passwordless.Example.csproj", "{15D70E7A-D222-4C60-93B2-06570A2BE5F2}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Passwordless.AspNetIdentity.Example", "examples\Passwordless.AspNetIdentity.Example\Passwordless.AspNetIdentity.Example.csproj", "{F9487727-715D-442F-BE2F-7FB9931606C2}"
```

Comment:
Maybe we should call the project `.AspNetCore.` to match the naming convention we adopted in the SDK? Or should we try to move away from that?